### PR TITLE
Code cleanup in projectmanager

### PR DESCRIPTION
**Pull request by hi-rustin**
Clean up projectmanager useless code and fix typos.

### CODE REVIEW

File: prow/plugins/projectmanager/projectmanager.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ const (
 
 var (
 	failedToAddProjectCard = "Failed to add project card for the issue/PR"
-	issueAlreadyInProject  = "The issue/PR %d already assigned to the project %s"
+	issueAlreadyInProject  = "The issue/PR %s already assigned to the project %s"
```
Comment (petr-muller) [inline prow/plugins/projectmanager/projectmanager.go:38]: This looks like this should be a number?

File: prow/plugins/projectmanager/projectmanager_test.go [hunk 2]
```diff
@@ -747,15 +747,15 @@ func checkCards(expectedColumnCards, projectColumnCards map[int][]github.Project
...
+				return fmt.Errorf("unable to find project card: %v under column: %d", card, columnID)
 			}
 		}
 	}
 	return nil
 }
 
 func TestHelpProvider(t *testing.T) {
-	var i int = 0
+	var i = 0
```
Comment (petr-muller) [inline prow/plugins/projectmanager/projectmanager_test.go:758]: nit: `var i int` is IMO a bit more idiomatic

File: prow/plugins/projectmanager/projectmanager.go (hunk 1 above)
Comment (petr-muller) [reply prow/plugins/projectmanager/projectmanager.go:38]: Ah ok, it's a URL. Nevermind.

**Review by petr-muller — APPROVED** [review]

**Review by fejta — APPROVED** [review]
/hold

**Comment by petr-muller** [comment]
@hi-rustin also, a suggestion - calling code "useless" is a little loaded. Not entirely hostile, but it is still an unnecessary judgement. I would suggest to use a little bit neutral "unused", or simply call it "Code cleanup in projectmanager" or something similar.

File: prow/plugins/projectmanager/projectmanager_test.go (hunk 2 above)
Comment (fejta) [reply prow/plugins/projectmanager/projectmanager_test.go:758]: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style is the source I cite for this

**Comment by fejta** [comment]
> I would suggest to use a little bit neutral "unused", or simply call it "Code cleanup in projectmanager" or something similar.

"Simplify unnecessary `(foo) to foo`" is how I would bikeshed the wording. I'd consider the differences in English between unused/unnecessary/useless to be pretty subtle :)

File: prow/plugins/projectmanager/projectmanager_test.go (hunk 2 above)
Comment (hi-rustin) [reply prow/plugins/projectmanager/projectmanager_test.go]: Fixed

**Comment by hi-rustin** [comment]
@petr-muller  @fejta  Thanks for your review. I changed the commit message to `Code cleanup in projectmanager`.

**Review by petr-muller — APPROVED** [review]
/hold cancel

**Pull request merged**